sql/config_test.go

```diff
@@ -20,14 +20,60 @@ package sql_test
 import (
 	"reflect"
 	"testing"
+	"time"
 
+	"github.com/cockroachdb/cockroach/client"
 	"github.com/cockroachdb/cockroach/config"
 	"github.com/cockroachdb/cockroach/keys"
 	"github.com/cockroachdb/cockroach/roachpb"
+	"github.com/cockroachdb/cockroach/server"
+	"github.com/cockroachdb/cockroach/sql"
+	"github.com/cockroachdb/cockroach/util"
 	"github.com/cockroachdb/cockroach/util/leaktest"
 	"github.com/gogo/protobuf/proto"
 )
 
+var configID = sql.ID(1)
+var configDescKey = sql.MakeDescMetadataKey(keys.MaxReservedDescID)
+
+// forceNewConfig forces a system config update by writing a bogus descriptor with an
+// incremented value inside. It then repeatedly fetches the gossip config until the
+// just-written descriptor is found.
+func forceNewConfig(t *testing.T, s *server.TestServer) (*config.SystemConfig, error) {
+	configID++
```

Comment:
nit: seems like you could just update the name each time instead of filling the ID space with junk?